backend/controllers/employerController.js

```diff
@@ -63,7 +57,7 @@ const registerEmployer = async (req, res) => {
 };
 
 const deleteEmployer = asyncHandler(async (req, res) => {
-    await employer.findOneAndDelete({ "_id": req.params.id }, (err, employer) => {
+    await Employer.findOneAndDelete({ "_id": req.params.id }, (err, employer) => {
         if (err) {
             res.status(500).json({ message: 'Error deleting user ' });
```

Comment:
Should keep error msgs consistent with other functions in this file. "Error deleting Employer"